lib/common.sh

```diff
@@ -246,28 +252,40 @@ export IRONIC_CLIENT_IMAGE=${IRONIC_CLIENT_IMAGE:-"${CONTAINER_REGISTRY}/metal3-
 export MARIADB_IMAGE=${MARIADB_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/mariadb"}
 export IRONIC_DATA_DIR="$WORKING_DIR/ironic"
 export IRONIC_IMAGE_DIR="$IRONIC_DATA_DIR/html/images"
-export IRONIC_KEEPALIVED_IMAGE=${IRONIC_KEEPALIVED_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/keepalived"}
 export IRONIC_NAMESPACE=${IRONIC_NAMESPACE:-"baremetal-operator-system"}
 export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
 
-# CAPM3 and IPAM controller images
+# BMO Release and CAPM3, BMO, IPAM images
```

Comment:
Can we explain these vars I noticed `BARE_METAL_OPERATOR_TAG ` `BMORELEASE` have same values and also the latter can be confused to `BMOBRANCH` a comment describing the each one why needed from where get its value and where used would help avoid conflicting then in the future

Reply:
maybe `BMOTAG` instead of `BARE_METAL_OPERATOR_TAG` is better ?

Reply:
Don't want to rename an existing var name since there are some other PRs tackling renaming. For now BMORELEASE and BARE_METAL_OPERATOR_TAG has the same value. Then again I didnt use same var for clarity. We are missing documentation for the all the Release vars. Lets do that in a followup.